avatar+badge/avatar.css

```diff
@@ -0,0 +1,135 @@
+@import "../for-import/reset.css";
+@import "../for-import/colors.css";
+
+.avatar-round {
+    border-radius: 50%;
+}
+
+.avatar-square-lg {
+    border-radius: 15px;
+}
+
+.avatar-square-md {
+    border-radius: 10px;
+}
+
+.avatar-square-sm {
+    border-radius: 5px;
+}
+
+.avatar-large {
```

Comment:
kindly use rems throughout the file so that while using media queries it would be easier to make page responsive . Use pixels only when the size is less than 10px